doc/topics/variables-extending-and-substitutions.rst

```diff
@@ -123,8 +123,8 @@ is typically used to set up a shared egg or cache directory, as in:
 .. code-block:: ini
 
   [buildout]
-  eggs-directory = ~/.buildout/eggs
-  download-cache = ~/.buildout/download-cache
+  eggs-directory = eggs
+  download-cache = download-cache
```

Comment:
In retrospect, I liked the explicit `~/.buildout/` version more -- it's more clear where it keeps the caches, and also it demonstrates that you can use `~` to refer to your home directory.